src/types/interface.ts

```diff
@@ -49,6 +51,54 @@ export interface InterfaceTypeOptions {
     // TODO: allowUnknownProperties?: boolean; // default: false
 }
 
+export interface InterfaceTypeOptionsWithPartial extends InterfaceTypeOptions {
+    /** Mark all properties as optional in this type. */
+    partial?: boolean;
+}
+
+export interface InterfaceMergeOptions {
+    /** The optional name for the type, uses a default TypeScript-like name if no name is given. */
+    name?: string;
+
+    /**
+     * Suppress the error about existing custom validations on one of the types that is being merged.
+     *
+     * @remarks
+     * Validations are reused when merging multiple interface types using {@link InterfaceType.withOptional | withOptional},
+     * {@link InterfaceType.withRequired | withRequired} and {@link InterfaceType.mergeWith | mergeWith}. But only when no properties
+     * overlap between the types that are being merged. As long as the properties don't overlap we can be sure that the merged type is
+     * assignable to each of the original types (`A & B` is assignable to both `A` and `B`). Therefore, the validations are still safe to
+     * run, even though the type has been extended with additional properties.
+     *
+     * However, when overlap is detected in the property-names of the types and any custom validation is encountered by Skunk Team types, an
+     * Error will be thrown. Use this option to ignore the custom validations and continue with the merge. Beware that all custom
+     * validations are then NOT applied to the resulting type.
+     */
+    ignoreValidations?: true;
```

Comment:
Because of the change in functionality mentioned above, maybe something like `skipValidations`/`omitValidations`..

Reply:
I also renamed `ignoreParsers` to `omitParsers` to keep that symmetrical, even though you don't have any other option than to omit the parsers.